srfi-234.html

```diff
@@ -65,30 +66,66 @@ <h2 id="specification">Specification</h2>
 <code>(</code><em>node dependent dependent</em> ...<code>)</code>,
 meaning that <em>dependents</em> are dependent on <em>node</em>.
 The optional argument <em>=</em> specifies the identity relation
-between nodes; it defaults to <code>eqv?</code>.</p>
+between nodes; it defaults to <code>equal?</code>.</p>
+
+<p>Returns the list of topologically sorted nodes or <code>#f</code>
+if the graph cannot be sorted topologically.</p>
+
+<p>If <em>graph</em> is circular, <code>topological-sort</code> may
```

Comment:
In the abstract, you say that "an error will be signaled," but here, you say that you return multiple values upon error.

Reply:
Since you have both options (i.e. through toplogical-sort/exception), consider mentioning that in the abstract.

Reply:
I mentioned that now — it made the abstract longer but I think that’s worth it (and it now includes the central procedure name in the abstract)